src/mcts/search.cc

```diff
@@ -111,6 +111,11 @@ class MEvaluator {
     const float child_m = child.GetM(parent_m_);
     float m = std::clamp(m_slope_ * (child_m - parent_m_), -m_cap_, m_cap_);
     m *= FastSign(-q);
+    if (q_threshold_ > 0.0f && q_threshold_ < 1.0f) {
```

Comment:
Could you add a comment what `GetM` function does?

I though it returns moves left, but then it returns 0 if |q| < q_thredshol, so it's not clear what it's supposed to do.

Reply:
It returns the "moves left head effect" term which gets added to the PUCT score, so that `PUCT = Q + U + M`. This PR doesn't change the meaning of `GetM()`, but your comment makes clear that it was already confusing before this PR, so a comment (even if it has nothing to do with this PR) would certainly justified.

Reply:
Something like this before the function declaration.
```
// Returns adjustment to Q based on Q and M value of a node.
```
Could you add this in this PR?

Also, `GetM()` sounds like a wrong name for this function. Maybe rename to `GetQAdjustment()` ?
(it's more or less clear that it's based on M because it inside the MEvaluator class).

Reply:
Settled at `GetMUtility()` because it best describes both where it comes from (M) and what it is (a utility).